backend/clubs/serializers.py

```diff
@@ -45,6 +45,28 @@
 from clubs.utils import clean
 
 
+ALL_TAGS_SELECTED_ERROR_MESSAGE = (
+    (
+        "We noticed you selected all of the options"
```

Comment:
You will be missing spaces in this text. You will need to manually add a space at the end (`of the options"` => `of the options "`).